bundles/org.openhab.binding.knx/src/main/java/org/openhab/binding/knx/internal/handler/KNXBridgeBaseThingHandler.java

```diff
@@ -65,16 +69,30 @@ public SecureRoutingConfig() {
         public long latencyToleranceMs = 0;
     }
 
+    /**
+     * Helper class to carry information which can be used by the
+     * command line extension (openHAB console).
+     */
+    public static class CommandExtensionData {
+        public CommandExtensionData() {
+            unknownGA = new TreeMap<>();
+        }
+
+        public Map<GroupAddress, BigDecimal> unknownGA;
+    }
+
     protected ConcurrentHashMap<IndividualAddress, Destination> destinations = new ConcurrentHashMap<>();
     private final ScheduledExecutorService knxScheduler = ThreadPoolManager.getScheduledPool("knx");
     private final ScheduledExecutorService backgroundScheduler = Executors.newSingleThreadScheduledExecutor();
     protected SecureRoutingConfig secureRouting;
     protected SecureTunnelConfig secureTunnel;
+    public CommandExtensionData commandExtensionData;
```

Comment:
this should be private and accessed from a getter (`public CommandExtensionData getCommandExtensionData()`).